addons/project/models/project.py

```diff
@@ -1178,6 +1182,13 @@ def _inverse_personal_stage_type_id(self):
         for task in self:
             task.personal_stage_id.stage_id = task.personal_stage_type_id
 
+    @api.depends('date_last_stage_update')
+    def _compute_days_from_last_stage_update(self):
+        for task in self:
+            task.days_from_last_stage_update = False
+            if task.date_last_stage_update:
+                task.days_from_last_stage_update = (fields.Datetime.now() - task.date_last_stage_update).days
```

Comment:
Normally, it is equivalent to: 
```suggestion
            task.days_from_last_stage_update = task.date_last_stage_update and (fields.Datetime.now() - task.date_last_stage_update).days
```